pages/profile/[username].tsx

```diff
@@ -100,12 +101,16 @@ const UserProfile: React.FC = () => {
       },
       {}
     )
+    let starsReceived = [] as Star[]
```

Comment:
i think its better to use 
`let starsReceived: Star[] | undefined`
than
` let starsReceived = [] as Star[]`
that way you don't have declare types for the same variable twice

Reply:
I think its okay to do `let starsReceived = [] as Star[]`, because then this always guarantees that the return object's `starsReceived` key is always array.